torch/nn/quantized/modules/wrapper_module.py

```diff
@@ -0,0 +1,48 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import torch
+from torch._ops import ops
+from torch.nn.modules import Module
+
+_FLOAT_MODULES = {
+    torch.add: ops.quantized.add
+}
+
+class WrapperModule(torch.nn.Module):
+    def __init__(self, operation):
+        super(WrapperModule, self).__init__()
+        self.operation = operation
+
+    def forward(self, x):
+        return self.operation(*x)
```

Comment:
cool, didn't know our JIT is smart enough to resolve `self.operation` (original reason to ask for explicit module)

why are you unpacking the args though? it's confusing to user and doesn't work with JIT (as it doesn't support varargs).

I guess we can just hard-code 2 arguments then or have per-num-arguments versions.

Reply:
I am not 100% sure if we will support unary ops through this mechanism. This is more of a "generic" implementation.

Reply:
I think it works with variable length, but only when using `torch.jit.trace`

Reply:
I'd rather make it work with Script too. You can create a two-argument version for now and hide it under `make_wrapper` function. In future, it'd be possible to extend to more arguments if needed (by inspecting schema of the passed operation)

Reply:
We don't have the `make_wrapper` anymore -- it's only a wrapper class :/

Reply:
again, I think having them scriptable is important - and starting with two-argument ops is fine. You could introduce `make_wrapper` to allow future insertion point for other number of arguments